lib/rabbit_feed/event.rb

```diff
@@ -2,42 +2,52 @@ module RabbitFeed
   class Event
     include ActiveModel::Validations
 
-    attr_reader :schema, :payload
-    validates_presence_of :schema, :payload
+    attr_reader :schema, :payload, :metadata
+    validates :metadata, presence: true
```

Comment:
as said before just making sure that events can not be created in an inconsistent state (not having a name)